### ISSUE TITLE
[Windows Terminal - Help]: No keyboard shortcut is provided for the 'About' option.

### ISSUE THREAD

**Issue by Saurabh220304** [microsoft/terminal#20594]
### Windows Terminal version

1.25.1912.0

### Windows build number

28120.2738

### Other Software

**Test Environment:**
OS Version: Windows 11 version 26H1 (OS Build 28120.2738)
App Name: Windows Terminal Preview
Screen Reader: Narrator

### Steps to reproduce

**Repro Steps:**
1.	Launch the 'Windows Terminal' application. 
2.	Press 'Ctrl + Shift + Spacebar' keys. 
3.	The application menu will open. 
4.	Navigate to the 'About' option. 
5.	Observe whether a keyboard shortcut is provided for the 'About' option.

**User Experience:**
Keyboard-only users need to navigate through the application menu to access the 'About' option, which makes accessing the feature less efficient.

**Attachment:**
https://onedrive.cloud.microsoft/:u:/a@rc32u9qu/S/IQBG9vfz8FjpRJUKtasuXq6uAQN_NSrKiAGCflvwb1A41QM?e=23LmF2

### Expected Behavior
A keyboard shortcut should be provided for the 'About' option so that keyboard users can quickly access it without navigating through the application menu.

### Actual Behavior
No keyboard shortcut is provided for the 'About' option.

**Comment by jeyaprasad** [comment]
Hi! I'd like to work on this issue. I'll investigate the existing keyboard shortcut handling for the Help menu and implement the required shortcut for the **About** option. I'll submit a PR once the changes are ready.

**Comment by zadjii-msft** [comment]
Hold up.

[Open about dialog](https://learn.microsoft.com/en-us/windows/terminal/customize-settings/actions#open-about-dialog)

There's an action for that. It's not bound to a keybinding by default. I really don't think it needs to be. Case in point:

<img width="723" height="267" alt="Image" src="https://github.com/user-attachments/assets/1dc2994c-10e9-4153-b104-f43dfc437623" />

Edge doesn't bind "About" to anything, and we're probably more conservative with our default keybindings than they are. 

IMO we should just reject this.
